code/modules/clothing/gloves/insulated.dm

```diff
@@ -0,0 +1,122 @@
+/obj/item/clothing/gloves/color
+	dying_key = DYE_REGISTRY_GLOVES
+	greyscale_colors = null
+
+/obj/item/clothing/gloves/color/yellow
```

Comment:
Did you intentionally leave ``color`` path here?

Reply:
yes cause they are yellow

Reply:
its stupid but whatever didnt want to repath everything int he game